src/main/java/lutchat/Storage.java

```diff
@@ -0,0 +1,107 @@
+package lutchat;
+
+import java.io.File;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
```

Comment:
You could add a blank line here to separate the java.io imports from the java.util imports according to the coding conventions. It would also make the import packages more readable if they were in alphabetical order.